repository-data/webfiles/src/main/resources/site/freemarker/include/pagination-nhs.ftl

```diff
@@ -0,0 +1,44 @@
+<#-- @ftlvariable name="pageable" type="org.onehippo.cms7.essentials.components.paging.Pageable" -->
+<#include "../include/imports.ftl">
+<#if pageable??>
+    <#if pageable.totalPages gt 1>
+        <nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+            <ul class="nhsuk-list nhsuk-pagination__list">
+                <#if pageable.previousPage??>
+                    <@hst.renderURL var="pageUrlPrevious">
+                        <@hst.param name="page" value="${pageable.previousPage}"/>
+                        <@hst.param name="pageSize" value="${pageable.pageSize}"/>
+                    </@hst.renderURL>
+                    <li class="nhsuk-pagination-item--previous">
+                        <a class="nhsuk-pagination__link nhsuk-pagination__link--prev" href="${pageUrlPrevious}">
+                            <span class="nhsuk-pagination__title">Previous</span>
```

Comment:
Should the text be maintained as CMS resource bundle instead of plain text ?

Reply:
Yes, you are right. I have updated the templates.